parlai/crowdsourcing/tasks/acute_eval/README.md

```diff
@@ -19,6 +19,9 @@ following BibTex entry:
     }
 
 # Code Instructions
+
+**NOTE: this code is incompatable with the latest version of Mephisto, beginning with the changes made in [PR #246](https://github.com/facebookresearch/Mephisto/pull/246). This task will be upgraded shortly to regain compatibility with the latest version; in the meantime, please use the version of Mephisto prior to that PR, at commit 99d19c91a69c035b413b68cefcbdbd42565e949a.**
```

Comment:
You can also refer people to the [`v0.1` tag](https://github.com/facebookresearch/Mephisto/releases/tag/v0.1)

Reply:
Good idea! Changed